src/gstreamer/gstkvssink.cpp

```diff
@@ -1609,6 +1609,16 @@ gst_kvs_sink_change_state(GstElement *element, GstStateChange transition) {
         case GST_STATE_CHANGE_READY_TO_PAUSED:
             gst_collect_pads_start (kvssink->collect);
             break;
+        default:
```

Comment:
This is great, at the start of this function can you add a comment block explaining the change, so it's visible to someone browsing the code itself, in addition to the PR where I see it's already added.

Reply:
Done!